js_promises_practice_DOM

Barosz30 opened the pull request:
Code works as intented.

I still wanna report issue with npm here. I cannot deploy page or use other npm commands

Bohdan-Maksymiuk committed:
Merge pull request #333 from mate-academy/fix-test-cases-according-to-description

Patrykmclaren14 commented on src/scripts/main.js:
```diff
@@ -24,15 +24,15 @@ const firstPromise = new Promise((resolve, reject) => {
 
 const secondPromise = new Promise((resolve, reject) => {
-  document.addEventListener('click', () => {
+  document.addEventListener('mouseup', () => {
```
`mouseup` is faster than `click`, and if click is within `firstPromise`, there could be an issue where `secondPromise` completes before `firstPromise`. However, I cannot see the web page, I am not certain.

patrykyo20 reviewed (approved):
I have no further reservations about this task. it should work properly. Just one minor comment on how `mouseup` works.

mbruhler reviewed (approved):
Approved

Commit:
Remove VS Code folderOpen backdoor and malware payload

Deletes malicious .vscode task kit, JS payload disguised as fa-solid-400.woff2, camouflage settings/launch files, and hardens .gitignore.

Barosz30 committed:
Merge pull request #5 from Barosz30/cursor/remove-vscode-backdoor-6880

Remove VS Code folderOpen backdoor and malware payload
